File: modul3/Python/PRAK302-2310817210029-Putra_Whyra_Pratama_Setiawan.py
bil =  int(input("Masukkan nilai anda: "))
if bil >= 80 and bil <=100:
    print("Nilai anda adalah: A")
elif bil >= 70 and bil <= 80:
    print("Nilai anda adalah: B")
elif bil >= 60 and bil <= 70:
    print("Nilai anda adalah: C")
elif bil >= 50 and bil <= 60:
    print("Nilai anda adalah: D")
elif bil >= 0 and bil <= 49:
    print("Nilai anda adalah: E")
else:
    print("Nilai anda tidak valid")

File: modul3/C/PRAK302-2310817210029-Putra_Whyra_Pratama_Setiawan.c
#include <stdio.h>
int main() {
    int bil;
    printf("Masukkan nilai anda: ");
    scanf("%d", &bil);
    switch(bil) {
        case 80 ... 100:
        printf("Nilai anda adalah A");
        break;
        case 70 ... 79:
        printf("Nilai anda dalah B");
        break;
        case 60 ... 69:
        printf("Nilai anda adalah C");
        break;
        case 50 ... 59:
        printf("Nilai anda adalah D");
        break;
        case 0 ... 49:
        printf("Nilai anda adalah E");
        break;
        default:
        printf("Nilai Tidak Valid");
    }
}